parcel/ParcelModel.jl

```diff
@@ -38,6 +38,9 @@ Base.@kwdef struct parcel_params{FT} <: CMP.ParametersType{FT}
     sampling_interval = FT(1)
     γ = FT(1)
     ip = CMP.Frostenberg2023(FT)
+    Nₐ = FT(1e8)
+    qₗ₀ = FT(1e-3)
```

Comment:
Could we either hide them in MixedModes size distribution parameters, or change the names to indicate that those parameters are only used for that one distribution type?